pkgs/applications/science/machine-learning/weights/keras/common.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenvNoCC, fetchurl, model, version, weights, sha256 }:
+
+stdenvNoCC.mkDerivation rec {
+  inherit version;
+  name = "${model}-${version}";
+  src = fetchurl {
+    inherit sha256;
+    url = "https://github.com/keras-team/keras-applications/releases/download/${model}/${weights}";
```

Comment:
The upstream tag is just `resnet` apparently so if your `${model}` is `resnet50` you will fail to fetch anything.
Moreover, I don't what upstream intend to do in the future but if they keep the same tag for future data updates then it means this URL is not stable. :/ (i.e. they may not use another tag for newer releases)

Reply:
I'm not sure what you mean here? Is there something you want to change here?